public/data_load/kibana_cloned_code/courier.ts

```diff
@@ -193,14 +193,15 @@ export const handleCourierRequest = async ({
     );
   }
 
-  inspectorAdapters.data.setTabularLoader(
+  // DataAdapter is not present in Kibana 7.12. Is this necessary?
+/*   inspectorAdapters.data.setTabularLoader(
     () =>
       buildTabularInspectorData((searchSource as any).tabifiedResponse, {
         queryFilter: filterManager,
         deserializeFieldFormat: getFieldFormats().deserialize,
       }),
     { returnsFormattedValues: true }
-  );
+  ); */
```

Comment:
Well, courier.ts is the mirror of this file:
https://github.com/elastic/kibana/blob/7.12/src/plugins/data/common/search/expressions/esaggs/request_handler.ts

So I invite you to take inspiration from this file.
This should fix "Inspect" feature, where "View: Data" does not work (it is empty).

Reply:
Please, see [this comment](https://github.com/fbaligand/kibana-enhanced-table/pull/230#discussion_r748653607)